src/main/java/com/squareup/javapoet/TypeSpec.java

```diff
@@ -126,7 +130,11 @@ void emit(CodeWriter codeWriter, String enumName, Set<Modifier> implicitModifier
         codeWriter.emitJavadoc(javadoc);
         codeWriter.emitAnnotations(annotations, false);
         codeWriter.emitModifiers(modifiers, Util.union(implicitModifiers, kind.asMemberModifiers));
-        codeWriter.emit("$L $L", kind.name().toLowerCase(Locale.US), name);
+        if (isAnnotationDeclaration()) {
```

Comment:
I wanted use `kind.name().toLowerCase()` too but we can't have a field which begins with an '@' in an enumeration.